lib/session.js

```diff
@@ -299,7 +299,7 @@ class Session {
     if (!stray.length) {
       return;
     }
-    cli.log(`Found stray commits in ${branchName}:\n` +
+    cli.log(`Found new commits in ${branchName}:\n` +
```

Comment:
Maybe:

```suggestion
    cli.log(`${branch} is out of sync with ${branchName}. ` +
            'Mismatched commits:\n' +
```

would be clearer